science/cdo/Portfile

```diff
@@ -105,3 +106,8 @@ variant cdi description {build, install and link to a CDI library} {
 livecheck.type              regex
 livecheck.url               ${homepage}
 livecheck.regex             {Version ([0-9]+\.[0-9]+\.[0-9]+\.*[0-9]*)}
+
+test.run                    yes
+test.cmd                    cd test && ${build.cmd}
```

Comment:
Instead of modifying `test.cmd` just to change the directory, please set `test.dir ${worksrcpath}/test`.

Reply:
Also, we typically leave livecheck settings as the last thing in the portfile. Settings related to a phase should be placed approximately chronologically into the portfile. So, the `patchfiles`  line you added above should go right above the `configure.args` line (with a blank line in between), since the patch phase occurs right before the configure phase, and the `test` lines should go before the `post-destroot` block.

Reply:
Thanks @ryandesign for your comments. I'm still learning since moving from developing Fink packages :)
* I was not aware of the `test.dir` setting. It is not mentioned in the MacPorts Guide (https://guide.macports.org/#reference.phases.test). I have adjusted the Portfile to use it.
* Nice that MacPorts has a preferred organisation of the Portfile. I've updated it as per your suggestion.
* I've also renamed the patchfile to follow the `patch-*.diff` preferred syntax.

See commit 790dedd.
I hope it can make it now through approval.